app/server/utils/transfer_limits.py

```diff
@@ -82,52 +87,48 @@ def token_is_liquid_type(credit_transfer):
     return credit_transfer.token and credit_transfer.token.token_type is token.TokenType.LIQUID
 
 
+def token_is_reserve_type(credit_transfer):
+    return credit_transfer.token and credit_transfer.token.token_type is token.TokenType.RESERVE
+
+
 def transfer_is_agent_out_subtype(credit_transfer):
     return credit_transfer.transfer_subtype is TransferSubTypeEnum.AGENT_OUT
 
 # ~~~~~~COMPOSITE CHECKS~~~~~~
 
 
-def is_liquid_and_has_foo(ct):
-    return token_is_liquid_type(ct) and ct.is_foo==True
+def is_user_and_any_token(credit_transfer):
```

Comment:
why can't we just use 'is_user' directly? feels like "is any token" would always be true?

Reply:
we want to apply Sempo Limits to any reserve and liquid token transactions. Might be weird cases were no token is defined, seemed reasonable to me